{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -20,17 +23,26 @@ before_install:
   - lsb_release -a
 install:
   - pip install tox
+    {%- if cookiecutter.test_runner|lower == "pytest" %} pytest-travis-fold{% endif %}
```

Comment:
Damn, this is my fault, moreover not the first time. I must have RTFM my own plugin. :disappointed:
Thanks!